### ISSUE TITLE
Cutting releases v1.9 and v2

### ISSUE THREAD

**Issue by edorivai** [ReactTraining/react-media#98]
Hey @mjackson :wave: 

**tl;dr**

- I think `master` is ready to be published as 1.9.
- Could you please review the changes in `next`?
- If you've given your :+1: for `next`, I'll go and polish the docs

A bit of an update on the current state of the repo. I've merged most of the outstanding PRs into either `master` or `next`. I think `master` should be publishable as a minor version bump, as we've only added Typescript definitions (#76) and the onChange callback (#95). Docs look good, and I've prepared the `CHANGES.md`, you just have to add the release date.

I don't think `next` is ready for deployment yet, as I'd like to do a cleanup of the docs. We've accumulated some new features over time, and while they've been added to the docs, I feel the docs now lack cohesion. Moreover, there's [an outstanding PR](https://github.com/ReactTraining/react-media/pull/96) which should improve the state of SSR (mainly preventing nasty edge-case bugs).

After we complete those things on `next`, we could release a v2-alpha. What do you think?

**Comment by flq** [comment]
Hi there,
would have loved the TS signatures, but it looks like a lot of good things are outstanding (last release is 8 months ago) - have you create some alternative package where I can pick up the latest additions?

**Comment by edorivai** [comment]
@flq I have not created a for - if that is what you mean. As it stands releases are done by @mjackson, who's a busy man :sweat_smile:. I guess you could for this repo for yourself if you really have to. Otherwise, if the TS typings are all you're after, you can just copy/paste them into your own repo, or you could even create an entry in DefinitelyTyped if you feel like it.

**Comment by flq** [comment]
I’m sure he’s busy. At the end I settled for react-responsive, which pretty much seems to do the same - I mean this can also always be a viable possibility, to stop supporting one lib in favor of another that does the same such that efforts can be focused.

**Comment by tstirrat15** [comment]
I'd love to see v2 released as well.

**Comment by grsmvg** [comment]
A minor version bump shouldn't be too much work right? Would be nice to have the Safari event listener bug solved, which is merged a month ago, @mjackson .

**Comment by mjackson** [comment]
Hey @edorivai, I think I forgot to mention that I fully automated releases on this repo :) I will take a look at the changes and cut some new releases today, but let's sync up on how this is to be done going forward. I don't want to be the bottleneck here :)

**Comment by edorivai** [comment]
@mjackson Sounds good, I'll shoot you an email so we can take the conversation there.

**Comment by edorivai** [comment]
@grsmvg, @flq, 1.9 has been published to npm, it includes TS signatures and a fix for our safari bug. Sorry for the delay :sweat_smile:.

@tstirrat15 progress on v2 will be tracked in #110

**Comment by grsmvg** [comment]
Great, thanks a bunch! :)

**Comment by grsmvg** [comment]
I get a bunch of runtime errors with version 1.9.0

```
Error: Cannot find module "@babel/runtime/helpers/esm/inheritsLoose"

Module not found: Error: Can't resolve '@babel/runtime/helpers/esm/assertThisInitialized' in ...

Module not found: Error: Can't resolve '@babel/runtime/helpers/esm/defineProperty' in ...

Module not found: Error: Can't resolve '@babel/runtime/helpers/esm/inheritsLoose' in ...
```
